tests/content/input/14.preserves empty lines in code blocks.md

```diff
@@ -1,9 +1,9 @@
 ::container
-```js
+```ts
 function test() {
-
-  console.log("test");
+  const name: string = 'My Name';
+  console.log("name", name);
```

Comment:
I added a `:` character here in the test to ensure indentation was not impacted when detecting YAML property keys that have a trailing `:`